ksql-engine/src/test/java/io/confluent/ksql/physical/PhysicalPlanBuilderTest.java

```diff
@@ -291,10 +291,10 @@ public void shouldRekeyIfPartitionByDoesNotMatchResultKey() {
     assertThat(lines.length, equalTo(4));
     assertThat(lines[0], equalTo(" > [ SINK ] | Schema: [ROWKEY STRING KEY, COL0 BIGINT, COL1 STRING, COL2 "
         + "DOUBLE] | Logger: InsertQuery_1.S1"));
-    assertThat(lines[1],
-        equalTo("\t\t > [ REKEY ] | Schema: [ROWKEY STRING KEY, COL0 BIGINT, COL1 STRING, COL2 DOUBLE] "
-            + "| Logger: InsertQuery_1.S1"));
-    assertThat(lines[2], equalTo("\t\t\t\t > [ PROJECT ] | Schema: [ROWKEY STRING KEY, COL0 BIGINT, COL1 STRING"
+    assertThat(lines[2],
+        containsString("[ REKEY ] | Schema: [TEST1.ROWKEY STRING KEY, TEST1.ROWTIME BIGINT, TEST1.ROWKEY STRING, TEST1.COL0 BIGINT, TEST1.COL1 STRING, TEST1.COL2 DOUBLE] "
```

Comment:
Why is the schema now prefixed?

Reply:
It just uses the source schema which is prefixed, for some reason, you can look at the rest of this test class to see this. As long as the sink schema is the same it should be fine I think.

Changing this is out of scope, though we should probably do it.